src/qmapshack/gis/trk/CTrkToRteDialog.h

```diff
@@ -0,0 +1,83 @@
+/**********************************************************************************************
+    Copyright (C) 2021 Oliver Eichler <oliver.eichler@gmx.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#ifndef CTRKTORTEDIALOG_H
+#define CTRKTORTEDIALOG_H
+
+#include "gis/prj/IGisProject.h"
+#include "gis/CGisWorkspace.h"
+#include "ui_ITrkToRteDialog.h"
+#include <QDialog>
```

Comment:
Get rid of include hell: As far as I can see you only need:

```
#include "gis/prj/IGisProject.h"
#include "ui_ITrkToRteDialog.h"
```

Reply:
I could remove only #include <QDialog> in the *.h file, but removed several includes in CTrkToRteDialog.cpp.